src/config/resolve-config.js

```diff
@@ -52,6 +62,13 @@ function _resolveConfig(filePath, opts, sync) {
       return null;
     }
 
+    if (!!includeFilePath) {
```

Comment:
Can this just be `if (includeFilePath)`?

Reply:
fixed!